Source/WebCore/DerivedSources-input.xcfilelist

```diff
@@ -2041,3 +2042,4 @@ $(PROJECT_DIR)/xml/XPathResult.idl
 $(PROJECT_DIR)/xml/XSLTProcessor.idl
 $(PROJECT_DIR)/xml/xmlattrs.in
 $(PROJECT_DIR)/xml/xmlnsattrs.in
+${WEBCORE_DIR}/css/htmlSwitchControl.css
```

Comment:
This looks unnecessary?

Reply:
I think this happened during compilation:
```
### Merging .xcfilelists for WebCore/macosx/Debug
".xcfilelist" files tell the build system what files are consumed and produced by the "Run
Script" build phases in Xcode. At least one of these .xcfilelist files was out of date and
has been updated. You now need to restart your build.
Command PhaseScriptExecution failed with a nonzero exit code
```
Let me undo that and try again.

Thanks for the review!